Documentation/development-environment.md

```diff
@@ -0,0 +1,127 @@
+kubernetes/preinstall : Install packages requirements ------------------ 25.18s
+download : container_download | Download containers if pull is required or told to always pull (all nodes) -- 23.74s
+download : container_download | Download containers if pull is required or told to always pull (all nodes) -- 18.90s
+download : container_download | Download containers if pull is required or told to always pull (all nodes) -- 15.39s
+kubernetes/master : Master | wait for the apiserver to be running ------ 12.44s
+download : container_download | Download containers if pull is required or told to always pull (all nodes) -- 11.83s
+download : container_download | Download containers if pull is required or told to always pull (all nodes) -- 11.66s
+kubernetes/node : install | Copy kubelet from hyperkube container ------ 11.44s
+download : container_download | Download containers if pull is required or told to always pull (all nodes) -- 11.41s
+download : container_download | Download containers if pull is required or told to always pull (all nodes) -- 11.00s
+docker : Docker | pause while Docker restarts -------------------------- 10.22s
+kubernetes/secrets : Check certs | check if a cert already exists on node --- 6.05s
+kubernetes-apps/network_plugin/flannel : Flannel | Wait for flannel subnet.env file presence --- 5.33s
+kubernetes/master : Master | wait for kube-scheduler -------------------- 5.30s
+kubernetes/master : Copy kubectl from hyperkube container --------------- 4.77s
+[leseb@tarox kubespray]$
+[leseb@tarox kubespray]$
+[leseb@tarox kubespray]$ vagrant ssh k8s-01
+Last login: Fri Jan 12 09:22:18 2018 from 192.168.121.1
+
+[vagrant@k8s-01 ~]$ kubectl get nodes
+NAME      STATUS    ROLES         AGE       VERSION
+k8s-01    Ready     master,node   2m        v1.9.0+coreos.0
+k8s-02    Ready     node          2m        v1.9.0+coreos.0
+k8s-03    Ready     node          2m        v1.9.0+coreos.0
+k8s-04    Ready     node          2m        v1.9.0+coreos.0
+k8s-05    Ready     node          2m        v1.9.0+coreos.0
+k8s-06    Ready     node          2m        v1.9.0+coreos.0
+k8s-07    Ready     node          2m        v1.9.0+coreos.0
+k8s-08    Ready     node          2m        v1.9.0+coreos.0
+k8s-09    Ready     node          2m        v1.9.0+coreos.0
+```
+
+## Development workflow on the host
+
+Everything should happen on the host, your development environment will reside on the host machine NOT inside the virtual machines running the Kubernetes cluster.
+
+Now, please refer to [https://rook.io/docs/rook/master/development-flow.html](https://rook.io/docs/rook/master/development-flow.html) to setup your development environment (go, git etc).
+
+At this stage, Rook should be cloned on your host.
+
+From your Rook repository (should be $GOPATH/src/github.com/rook) location execute `bash tests/scripts/multi-node/build-rook.sh`.
+During its execution `build-rook.sh` also purge your Rook pods so Rook container images can be build.
```

Comment:
This wording is a bit awkward here, perhaps try the following (note i've included the suggestion to call the script `deploy-rook.sh`):
```
During it's execution, `deploy-rook.sh` will purge all running Rook pods from the cluster, so that your latest container image can be deployed.
Furthermore, **all Ceph data and config will be purged** as well.
Ensure that you are done with all existing state on your test cluster before executing `deploy-rook.sh` as it will clear everything.
```

Reply:
thanks